package.json

```diff
@@ -38,10 +38,9 @@
     "@babel/plugin-syntax-dynamic-import": "7.2.0",
     "@babel/traverse": "7.26.4",
     "@babel/types": "7.17.0",
-    "babel-plugin-flow-to-typescript": "git+https://github.com/Kiikurage/babel-plugin-flow-to-typescript.git#1375d207e9dd85e9e697cf3d90301171050ca77f",
```

Comment:
I was getting a `tsc` error when installing this and I'm pretty sure we don't need this since we don't support flow anywhere

so I got rid of this and ripped out [the usage](https://github.com/datadotworld/ts-migration/pull/26/files#diff-2c718087c1fd72979602bd9da34119fb956d070d90109b6bc01ea23b2ad7eae1R39)